### ISSUE TITLE
automatical deletion of persistence history file

### ISSUE THREAD

**Issue by Pythonaire** [simont77/fakegato-history#130]
If i'm right, uploading the data to the EVE app/homekit  is triggered by the user action. 
Using persistent storing in parallel can produce large files. I would like to delete these persistent storage files after uploading the containing data to the homekit automatically. 
How can i see in the code: "ok, all data are uploaded, the persistent data file is no longer needed."? 
I tried "this.currentEntry > this.lastEntry" as a trigger to call  the remove function, but without luck. 

Thx
